### PR TITLE
Fix panics in `scene_viewer` and `audio_control`

### LINKED ISSUE #16978: `scene_viewer` and `audio_control` examples panic
## Bevy version

main, bisected to #16638

## What you did

`cargo run --example scene_viewer`
`cargo run --example audio_control`

## What went wrong

```
thread 'main' panicked at /Users/me/src/bevy/crates/bevy_ecs/src/system/function_system.rs:216:28:
scene_viewer::camera_controller::run_camera_controller could not access system parameter Single<(&mut Transform, &mut CameraController), With<Camera>>
```

```
thread 'main' panicked at /Users/me/src/bevy/crates/bevy_ecs/src/system/function_system.rs:216:28:
audio_control::update_speed could not access system parameter Single<AudioSink, With<MyMusic>>
```

## Additional information

Related to #16959

### PR DESCRIPTION
**Pull request by rparrett**
# Objective

Fixes #16978

While testing, discovered that the morph weight interface in `scene_viewer` has been broken for a while (panics when loaded model has morph weights), probably since #15591. Fixed that too.

While testing, saw example text in morph interface with [wrong padding](https://bevyengine.org/learn/contribute/helping-out/creating-examples/#visual-guidelines). Fixed that too. Left the small font size because there may be a lot of morphs to display, so that seems intentional.

## Solution

Use normal queries and bail early

## Testing

Morph interface can be tested with
```
cargo run --example scene_viewer assets/models/animated/MorphStressTest.gltf
```

## Discussion

I noticed that this fix is different than what is happening in #16976. Feel free to discard this for an alternative fix. I opened this anyway to document the issue with morph weight display.

This is on top of #16966 which is required to test.

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]

File: examples/tools/scene_viewer/morph_viewer_plugin.rs [hunk 1]
```diff
@@ -196,7 +204,21 @@ fn update_text(
             target.weight = actual_weight;
         }
         let key_name = &AVAILABLE_KEYS[i].name;
-        *writer.text(*text, i + 3) = format!("[{key_name}] {target}\n");
```
[the remark this answers is not in the document]
Comment (rparrett) [reply examples/tools/scene_viewer/morph_viewer_plugin.rs:199]: It was broken in the PR referenced in the description and just panics now due to missing spans.

However, after looking closer at the PR that broke this, I see a better fix, so please stand byl

File: examples/audio/audio_control.rs [hunk 2]
```diff
@@ -34,29 +34,48 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
 #[derive(Component)]
 struct MyMusic;
 
-fn update_speed(sink: Single<&AudioSink, With<MyMusic>>, time: Res<Time>) {
+fn update_speed(music_controller: Query<&AudioSink, With<MyMusic>>, time: Res<Time>) {
+    let Ok(sink) = music_controller.get_single() else {
```
[the remark this answers is not in the document]
Comment (alice-i-cecile) [reply examples/audio/audio_control.rs:38]: `Single` now panics again. Using `Option<Single>` here would work, but that's kind of a sidegrade.

[the remark this answers is not in the document]
Comment (MiniaczQ) [reply examples/audio/audio_control.rs:38]: Or we can reconfigure the system to ignore failing params

[the remark this answers is not in the document]
Comment (rparrett) [reply examples/audio/audio_control.rs:38]: I personally feel like specifying `Single` and disabling warnings feels sort of like a dirty way to write this code, but please discuss and feel free to close for an alternative implementation.

Or I can fix it up late tonight or tomorrow if there's consensus the other way.

[the remark this answers is not in the document]
Comment (MiniaczQ) [reply examples/audio/audio_control.rs:38]: This solution is totally fine, just listing alternatives

**Comment by MiniaczQ** [comment]
Got this while running `scene_viewer`
```bevy_winit::accessibility::update_accessibility_nodes could not access system parameter Res<InputFocus>```

**Review by MiniaczQ — APPROVED** [review]
Examples seem fine, the accessibility issue isn't related to https://github.com/bevyengine/bevy/issues/16978
so I'm not pushing to fix it here, especially since there is no reproduction

File: examples/audio/audio_control.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (alice-i-cecile) [reply examples/audio/audio_control.rs:38]: Agreed, this solution is fine.

**Pull request merged**